$NetBSD: patch-ab,v 1.4 2011/05/13 23:25:14 mwdavies Exp $

dont install hicolor/index.theme as hicolor-icon-theme pkg will

--- pics/CMakeLists.txt.orig	2007-10-26 01:21:45.000000000 +1300
+++ pics/CMakeLists.txt
@@ -1,5 +1,5 @@
 
-add_subdirectory( hicolor )
+#add_subdirectory( hicolor )
 add_subdirectory( emoticons )
 #add_subdirectory( oxygen )
 
